velox/functions/prestosql/aggregates/MinMaxByAggregates.cpp

```diff
@@ -49,10 +49,12 @@ struct Comparator {
       // is less than vector value.
       if constexpr (greaterThan) {
         return !accumulator->hasValue() ||
-            prestosql::compare(accumulator, newComparisons, index) < 0;
+            functions::aggregate::compare(accumulator, newComparisons, index) <
```

Comment:
nit: do we need the prefix functions::aggregate as the namespace `facebook::velox::functions::aggregate` is used?

Reply:
I checked this is needed, thanks.

Reply:
@zhli1142015 Why this is needed? Seems strange.